internal/test/schema.go

```diff
@@ -0,0 +1,93 @@
+// Copyright 2021 Zenauth Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+//go:build tests
+// +build tests
+
+package test
+
+import (
+	"google.golang.org/protobuf/types/known/structpb"
+
+	schemav1 "github.com/cerbos/cerbos/api/genpb/cerbos/schema/v1"
+)
+
+// GenSchema generates a sample schema.
+func GenSchema() *schemav1.Schema {
```

Comment:
I see you're trying to follow the precedent of `GenPolicy` functions -- which is good. But, if you take a closer look, they are there to generate dynamic policies with variable number of policy rules whereas this `GenSchema` function simply produces a static schema. You could have easily just read it from a file instead and not spend so much effort on hand-crafting the protobufs 😄